README.md

```diff
@@ -31,44 +31,49 @@ nats.ws supports Promises, depending on the browser/runtime environment you can
 
 Load the library:
 ```html
-<script src='./nats.js'></script>
+<script src='node_modules/nats.ws/nats.js'></script>
```

Comment:
Actually the old reference is the correct reference is you forked and installed the repo (it should have compiled, and webpacked when you copied/installed). `node_modules` is just simply a dev artifact.